setup.cfg

```diff
@@ -54,15 +49,14 @@ install_requires =
     numpy==1.22.2; platform_machine=='loongarch64' and python_version<'3.11'
 
     # default numpy requirements
-    numpy==1.13.3; python_version=='3.5' and platform_machine not in 'aarch64|loongarch64' and platform_system!='AIX'
-    numpy==1.13.3; python_version=='3.6' and platform_machine not in 'aarch64|loongarch64' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
-    numpy==1.14.5; python_version=='3.7' and platform_machine not in 'arm64|aarch64|loongarch64' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
-    numpy==1.17.3; python_version=='3.8' and platform_machine not in 'arm64|aarch64|s390x|loongarch64' and platform_python_implementation != 'PyPy'
-    numpy==1.19.3; python_version=='3.9' and platform_system not in 'OS400' and platform_machine not in 'arm64|loongarch64' and platform_python_implementation != 'PyPy'
+    numpy==1.14.5; python_version=='3.7' and platform_machine not in 'arm64|aarch64|ppc64le|loongarch64' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
+    numpy==1.17.3; python_version=='3.8' and platform_machine not in 'arm64|aarch64|ppc64le|s390x|loongarch64' and platform_python_implementation != 'PyPy'
+    numpy==1.19.3; python_version=='3.9' and platform_system not in 'OS400' and platform_machine not in 'arm64|loongarch64'
     # Note that 1.21.3 was the first version with a complete set of 3.10 wheels,
     # however macOS was broken and it's safe to build against 1.21.6 on all platforms (see gh-28 and gh-45)
-    numpy==1.21.6; python_version=='3.10' and platform_machine!='loongarch64' and platform_python_implementation != 'PyPy'
-    numpy==1.23.2; python_version=='3.11' and platform_python_implementation != 'PyPy'
+    # we want to exclude win-arm64, but cannot write "not (...)" in this configuration format, thus use De Morgan's law
```

Comment:
I suggest deleting this comment or rephrasing to a simple "the last condition excludes win-arm64". I wasn't familiar with that law, and it's more confusing that helpful to reference it.

Reply:
More importantly: these are the default requirements, and I don't see win-arm64 for py310/py311 special-cased elsewhere. So this seems wrong. Or am I missing the point of this change?

Reply:
In all the previous conditions, win-arm64 is excluded by virtue of `arm64` being excluded. Now that osx-arm64 is not exceptional anymore, we need to distinguish the windows-case specifically.

> I suggest deleting this comment or rephrasing to a simple "the last condition excludes win-arm64". I wasn't familiar with that law, and it's more confusing that helpful to reference it.

I guess my math studies show a bit 😋
The rule is simple enough (`not (A & B) == (not A) or (not B)`), and I just gave it its "proper" name. I left a comment because I find the condition unintuitive to read, so I though it's better to note why it came about. Happy to adapt or remove it though.

Reply:
> In all the previous conditions, win-arm64 is excluded by virtue of `arm64` being excluded.

That's an accident, because the code is specifically written for macOS arm64 - which was the only platform to use `arm64` as an identifier. The special cases are for:
```
platform_machine=='arm64' and platform_system=='Darwin'
```
which is a single platform. So the intent in the default requirements is to exclude that same macOS arm64 condition. That is the right fix here. The current change for Python 3.10/3.11 must be wrong, you cannot exclude a platform from the defaults without including it elsewhere as a special case.

Reply:
> you cannot exclude a platform from the defaults without including it elsewhere as a special case.

I maintain that this is the correct approach if that platform doesn't have _any_ compatibe numpy version yet (that I'm aware of) - i.e. it falls back to plain `numpy`, and that's intentional.

Unless you tell me that someone has compiled and tested 1.25 (or whatever) for win-arm64 successfully, then we can set that.

Reply:
I doubt we go back to Python 3.7 & numpy 1.14.3, but I've just kept the defaults for win-arm64 now.